composer.json

```diff
@@ -26,7 +26,8 @@
     "require": {
         "php": "~5.4",
         "ext-openssl": "*",
-        "namshi/jose": "~4.0"
+        "namshi/jose": "~4.0",
+        "willdurand/negotiation": "1.3.4"
```

Comment:
I used to come down on the '~' side, and I think that makes sense for the dep on 'php', I'm no longer sure otherwise and have switched to just using fixed versions. I guess until we start seeing reports of problems we can go the '~' route to be consistent and because that seems common here, but I'm not opposed to setting all non-PHP dependencies to a single version. Frankly given we're testing on php 5.5 via Travis we could even switch it to a '^'. Not sure where that leaves us.